### ISSUE TITLE
Ceil clubbing

### ISSUE THREAD

**Issue by eightyeight** [GarageGames/Torque3D#1237]
Sepf-explanatory, really. https://github.com/Azaezel/Torque3D/commit/6b4bc85722898fe345d65faf4cabd733ed4cd2dd

**Comment by andr3wmac** [comment]
In @Azaezel's branch the problem is self-inflicted. In stock the range of the slider is 1 to 128 and this is the expected range of values for specular power. That value is packed into a single byte so mCeil does no harm and keeps the values cleaner when working with the slider. However, in @Azaezel's branch he changed the slider range to 0 to 1 so mCeil was ruining his range of values, so it makes sense to remove it. The change makes sense in his branch, but it's not a current issue in stock as far as I know.

**Comment by Azaezel** [comment]
Correct. PBR and similar expect scalars for their calculations. While it wouldn't particularly be *harmful* to knock that on out for stock, given there's no networking and the like requiring value constraints, no foul there aside from the ease of overlooking it when converting.
